usefuls.clj

```diff
@@ -0,0 +1,6 @@
+;(write-ast (parse-expr "int c = 1 + 3;"))
```

Comment:
oops, I was sure I rm'ed it before I commit, don't know why it showed up. So every time I try to push to my repo, the pull request gets updated?